eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/jobs/steps/processors/SubSnpNoHgvsToVariantProcessorTest.java

```diff
@@ -0,0 +1,380 @@
+                                                     Orientation.FORWARD, Orientation.REVERSE, Orientation.REVERSE,
+                                                     CONTIG_START, false, false, "A", CREATED_DATE, TAXONOMY);
+
+        List<DbsnpSubmittedVariantEntity> variants = processor.process(subSnpNoHgvs);
+
+        assertProcessedVariant(subSnpNoHgvs, variants.get(0), "T", "-");
+    }
+
+    @Test
+    public void transformVariantSupportedByEvidence() throws Exception {
+        SubSnpNoHgvs subSnpNoHgvs = new SubSnpNoHgvs(25945162L, 14730808L, "C/T", ASSEMBLY, BATCH_HANDLE, BATCH_NAME,
+                                                     CHROMOSOME, CHROMOSOME_START, CONTIG_NAME, DbsnpClass.SNV,
+                                                     Orientation.FORWARD, Orientation.REVERSE, Orientation.FORWARD,
+                                                     CONTIG_START, false, true, "G", CREATED_DATE, TAXONOMY);
+
+        List<DbsnpSubmittedVariantEntity> variants = processor.process(subSnpNoHgvs);
+
+        assertProcessedVariant(subSnpNoHgvs, variants.get(0), CHROMOSOME, CHROMOSOME_START, "G", "A", true, true);
+    }
+
+    @Test
+    public void transformMultiallelicSS() throws Exception {
+        SubSnpNoHgvs subSnpNoHgvs = new SubSnpNoHgvs(1975823489L, 13637891L, "A/C/G", ASSEMBLY, BATCH_HANDLE,
+                                                     BATCH_NAME, CHROMOSOME, CHROMOSOME_START, CONTIG_NAME,
+                                                     DbsnpClass.SNV, Orientation.FORWARD, Orientation.FORWARD,
+                                                     Orientation.FORWARD, CONTIG_START, false, false, "A", CREATED_DATE,
+                                                     TAXONOMY);
+
+        List<DbsnpSubmittedVariantEntity> variants = processor.process(subSnpNoHgvs);
+
+        assertProcessedVariant(subSnpNoHgvs, variants.get(0), "A", "C");
+        assertProcessedVariant(subSnpNoHgvs, variants.get(1), "A", "G");
+    }
+
+    @Test
+    public void transformMNV() throws Exception {
+        SubSnpNoHgvs subSnpNoHgvs = new SubSnpNoHgvs(666079762L, 313826846L, "TTA/ATC", ASSEMBLY, BATCH_HANDLE,
+                                                     BATCH_NAME, CHROMOSOME, CHROMOSOME_START, CONTIG_NAME,
+                                                     DbsnpClass.MVN, Orientation.FORWARD, Orientation.REVERSE,
+                                                     Orientation.REVERSE, CONTIG_START, false, false, "TAA",
+                                                     CREATED_DATE, TAXONOMY);
+
+        List<DbsnpSubmittedVariantEntity> variants = processor.process(subSnpNoHgvs);
+
+        assertProcessedVariant(subSnpNoHgvs, variants.get(0), "TTA", "ATC");
+    }
+
+    @Test
+    public void transformMultiallelicMNV() throws Exception {
+        SubSnpNoHgvs subSnpNoHgvs = new SubSnpNoHgvs(973513424L, 525103154L, "AC/TC/TA", ASSEMBLY, BATCH_HANDLE,
```

Comment:
AC/TC is not an MNV

Reply:
You are right, but Dbsnp says the variant class is MVN: https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?searchType=adhoc_search&type=rs&rs=rs525103154

The "Decluster SS is variant class does not match" is not exactly this (that is a SS that has a different class that the RS). @jmmut , @cyenyxe , what should we do here?

Reply:
and we didn't put unique constraints for the ss accession field in the dbsnp table, so it's allowed to have this:
{ hash_1, A>T, ss1, rs1}
{ hash_2, AC>TA, ss1 , rs=null}  
If I'm remembering everyting properly